core/ecschema-editing/src/Differencing/SchemaDiagnosticVisitor.ts

```diff
@@ -449,6 +452,27 @@ export class SchemaDiagnosticVisitor {
     }
     return;
   }
+
+  private visitMissingPresentationUnit(diagnostic: AnyDiagnostic) {
+    const koq = diagnostic.ecDefinition as KindOfQuantity;
+    const [presentationFormat] = diagnostic.messageArgs as [Format | OverrideFormat];
+
+    let modifyEntry = this.schemaItemDifferences.find((entry): entry is KindOfQuantityPresentationFormatDifference => {
```

Comment:
that should actually be an addEntry. I saw it's also wrong for the mixins on classes and relationship constrains. Looks like that slipped through when we changed the change type from modify to add there.